pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiWorkflowStudyRedirectState.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.ddp.db.dao;
+
+import java.util.Optional;
+
+import org.broadinstitute.ddp.model.workflow.StudyRedirectState;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.config.RegisterConstructorMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+public interface JdbiWorkflowStudyRedirectState extends SqlObject {
```

Comment:
Can we put these into `JdbiWorkflowState` so we have one less file to deal with? Even better, merge and remove `JdbiWorkflowActivityState` as well.